Automattic/jetpack
# Fix masterbar not being displayed entirely for resolutions of under 660px.

Fixes #19695

#### Changes proposed in this Pull Request:

* When the 660px media query is applied we won't apply any margin-left to the wpcontent element/container.

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:





For Atomic sites, install Jetpack beta plugin and switch to this branch
For Simple Sites, apply to your sandbox the following patch: D70569-code and D70600-code

* Go to a WP-Admin page.
* Resize your window to a width lower than 660px.
* Click on "My Sites" menu.
* The entire masterbar is displayed.

##### Before
<img width="467" alt="Captura de pantalla 2021-11-24 a las 17 03 17" src="https://user-images.githubusercontent.com/1989914/143273351-fdf6f22c-7e9e-45f9-9874-cae61ddd15c6.png">

##### Expected After
<img width="386" alt="Captura de pantalla 2021-11-24 a las 17 03 33" src="https://user-images.githubusercontent.com/1989914/143273397-b96742e3-c350-40fd-90af-188353e73c48.png">
